NuRadioReco/modules/correlationDirectionFitter.py

```diff
@@ -23,12 +24,11 @@ def __init__(self):
         self.__azimuth = []
         self.__delta_zenith = []
         self.__delta_azimuth = []
-        self.logger = logging.getLogger('NuRadioReco.correlationDirectionFitter')
         self.begin()
 
     def begin(self, debug=False, log_level=None):
         if(log_level is not None):
-            self.logger.setLevel(log_level)
+            logger.setLevel(log_level)
```

Comment:
can you change the logger back to `self.logger`. Also don't call `basicConfig()`.